src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Policy/UnconditionalPolicy.cs

```diff
@@ -1,10 +1,9 @@
 ﻿using CoreWCF.IdentityModel.Claims;
-using CoreWCF;
+using CoreWCF.Security;
```

Comment:
Why was CoreWCF.Security needing to be added when no code changes were made in the file?

Reply:
please refer to previous comment..if agree resolve